src/main/java/uk/nhs/digital/nhsconnect/nhais/mapper/PersonDateOfBirthMapper.java

```diff
@@ -17,6 +21,12 @@ public PersonDateOfBirth map(Parameters parameters) {
 
     private Instant getPersonDob(Parameters parameters) {
         Patient patient = ParametersExtension.extractPatient(parameters);
-        return patient.getBirthDate().toInstant();
+        var birthDateElement = patient.getBirthDateElement();
```

Comment:
Fhir parser sets local timezone when parsing json
I don't know how to change that so I did this trick